.github/workflows/openqa.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: openQA - GitHub action example
+# yamllint disable-line rule:truthy
+on:
+  pull_request:
+  issue_comment:
+    types: [created, edited]
+  workflow_dispatch:
+env:
+  OPENQA_TOKEN: ${{ secrets.OPENQA_API_USER }}:${{ secrets.OPENQA_API_KEY }}:${{ secrets.OPENQA_API_SECRET }}
+  OPENQA_HOST: ${{ secrets.OPENQA_URL }}
+
+jobs:
+  trigger_and_monitor_openqa:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.opensuse.org/home/okurz/container/ca/containers/tumbleweed:curl-jq-ssh-git
+    steps:
+      - uses: actions/checkout@v3
+      - name: Trigger and monitor openQA test
+        run: tools/trigger-openqa-jobs
```

Comment:
I know that a separate script looks like making it more clear to follow what is going on in there but the goal is to have a generic github action workflow that can be used in various projects. So we shouldn't add a script as part of this repo. Of course the pull request still serves as a valid example how the extended openQA API can be used.

Reply:
Maybe this script could live in another repo that can be included where needed? That everyone just C&P code makes not much sense but keeping it in `openqa.yml` makes this even worse because then the C&P code is mixed with other repo-specific config (like when it is triggered and names).

Reply:
Yes, I am not encouraging copy-pasting unless it's like one (reasonable long) line of code, not more. So either we need to make any calls to the openQA API so simple that everything fits in one line of code or there is a github action repo that can be used from any repo.